db/migrate/20211104161101_add_privileged_users_reaction_sum.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrivilegedUsersReactionSum < ActiveRecord::Migration[6.1]
+  def change
+    add_column :articles, :privileged_users_reaction_points_sum, :integer, default: 0
```

Comment:
sanity check: this is used in calculations/conditionals on objects/rows we already loaded by another means, and not for search, so we don't expect to need an index for this column?

Is this column nullable?

This migration is not able to rollback safely (the strong migrations gem appears to be at work).  I don't know if that's important, but worth noting. I usually check migrations with an up-down-up verification, and in this case specially I wanted to double check that the `:default` column value infilled existing rows during the creation since bin/setup had run the updates before I could see.
 

```
~/src/forem$ bundle exec rails db:rollback
== 20211104161101 AddPrivilegedUsersReactionSum: reverting ====================
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:


=== Dangerous operation detected #strong_migrations ===

Active Record caches attributes, which causes problems
when removing columns. Be sure to ignore the column:

class Article < ApplicationRecord
  self.ignored_columns = ["privileged_users_reaction_points_sum"]
end

Deploy the code, then wrap this step in a safety_assured { ... } block.

class AddPrivilegedUsersReactionSum < ActiveRecord::Migration[6.1]
  def change
    safety_assured { remove_column :articles, :privileged_users_reaction_points_sum, :integer, default: 0 }
  end
end
```

Reply:
Noted and updated.  Thanks for the details.

Reply:
for anyone from the future reading this comment - the strong migrations safety blocks these rollbacks - if you wanted or needed to roll the migration back locally, disable the safety check. 

Following the suggested "fix" when applying an up migration would add a column that's not accessible to the model (probably not what you wanted).

This is captured better in https://github.com/forem/forem/pull/15283#discussion_r743374991 and I'm just noting it before I resolve the conversation in case this is read first and it confuses anyone.